src/marqo/vespa/vespa_client.py

```diff
@@ -142,7 +143,7 @@ def wait_for_application_convergence(self, timeout: int = 120) -> None:
         raise VespaError(f"Vespa application did not converge within {timeout} seconds")
 
     def query(self, yql: str, hits: int = 10, ranking: str = None, model_restrict: str = None,
-              query_features: Dict[str, Any] = None, **kwargs) -> QueryResult:
+              query_features: Dict[str, Any] = None, timeout: float = None, **kwargs) -> QueryResult:
```

Comment:
this timeout is `float` while others are `int`

Reply:
Addressed